cpp/include/cudf/fixed_point/floating_conversion.hpp

```diff
@@ -34,6 +35,49 @@ namespace numeric {
 
 namespace detail {
```

Comment:
The entire source of this file is in the detail namespace but the file is in a public header directory.
Are these meant to be called only internally then? We are evaluating public/internal functions/source recently.
Perhaps this could be moved to detail directory instead -- `cudf/fixed_point/detail` would probably be ok.

Now I'm seeing this is called from public `unary.hpp` which has its own issue.
So this is probably out of scope for this PR.

Reply:
Is there an issue for this change?

Reply:
Some of this functionality needs to be exposed for the spark-rapids-jni repository to wrap around for it to perform conversions with the spark-specific rounding.  I'm still finalizing the details for the sr-jni code, when that's done I'll have a better idea of what the minimal amount of code we can expose is.